modules/smtp/site.js

```diff
@@ -510,16 +487,16 @@ $(function () {
                 modalContentHeadline = "Your subject and body are empty!";
             }
 
-            // if contact_cc not exist in contact list for user
-            var checkInList = "";
-            if (list_emails) {
-                checkInList = check_cc_exist_in_contacts_list(e);
+            if (typeof check_cc_exist_in_contacts_list === 'function') {
```

Comment:
This solution will not solve the issue, the goal is to be able to send an email even though module contact is no enabled, I think you should check if the module is enabled, if yes then active the feature, if no then we must be able to send an email.